tests/config/prettier-plugins/prettier-plugin-toml/index.js

```diff
@@ -0,0 +1,7 @@
+import createPlugin from "../../utils/create-plugin.cjs";
+
+const COMMENT = "/* Formatted by toml plugin */";
+export default createPlugin({
+  name: "toml",
+  print: (text) => COMMENT + "\n" + text.replace(COMMENT, "").trim(),
+});
```

Comment:
Since this is not a realy toml plugin, let name it as 

`prettier-plugin-dummy-toml`